bindings/python/gtn/_creations.cpp

```diff
@@ -18,19 +18,21 @@ using namespace py::literals;
 PYBIND11_MODULE(_creations, m) {
   m.def(
       "scalar_graph",
-      [](float weight, bool calcGrad) {
+      [](float weight, Device device, bool calcGrad) {
         py::gil_scoped_release release;
-        return scalarGraph(weight, calcGrad);
+        return scalarGraph(weight, device, calcGrad);
       },
       "weight"_a,
+      "device"_a = Device(DeviceType::CPU),
```

Comment:
nit: this can just be `Device::CPU` as it will implicitly construct into a `Device`.

Reply:
Evidently I'm wrong about this, it doesn't implicitly construct a `Device` here. I might add an overload so that you can pass `gtn.CPU` and `gtn.CUDA` directly.